sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CResourceServerAutoConfiguration.java

```diff
@@ -54,15 +52,15 @@ public AADB2CResourceServerAutoConfiguration(@NonNull AADB2CProperties propertie
 
     @Bean
     @ConditionalOnMissingBean
-    public AADTrustedIssuerRepository trustedIssuerRepository() {
-        return new AADTrustedIssuerRepository(properties.getTenantId());
+    public AADB2CTrustedIssuerRepository trustedIssuerRepository() {
```

Comment:
Change this to `AADTrustedIssuerRepository`, because we're coding with the base class.